website/react-magma-docs/src/pages/api/nav-tabs.mdx

```diff
@@ -187,6 +187,65 @@ export function Example() {
 }
 ```
 
+## Text Transform
+
+#### For all NavTabs
+
+Options for the `textTransform` prop for navtabs include `uppercase` and `none`, with `uppercase` (all caps) being the default value.
+This sets the CSS `text-transform` property.
+
+```tsx
+import React from 'react';
+import { NavTabs, NavTab, TabsTextTransform } from 'react-magma-dom';
+
+export function Example() {
+  return (
+    <>
+      <NavTabs textTransform={TabsTextTransform.uppercase}>
+        <NavTab isActive to="#">
+          First uppercase
+        </NavTab>
+        <NavTab to="#">Second uppercase</NavTab>
+      </NavTabs>
+      <br />
+      <br />
+      <br />
+      <NavTabs textTransform={TabsTextTransform.none}>
+        <NavTab isActive to="#">
+          First lowercase
+        </NavTab>
+        <NavTab to="#">Second lowercase</NavTab>
+      </NavTabs>
+    </>
+  );
+}
+```
+
+#### For a certain Tab
```

Comment:
@orion-cengage do we want to support this, or should this style be set the same for all NavTabs?

Reply:
Does this mean you have the ability to set it on all tabs at the top level, and then "override" that prop on a specific tab if necessary? I could see that being useful for something like:

DATABASE     PERIODICALS     eBOOK

Reply:
Never mind, I just saw it in action in the docs. I really like that!

![image](https://github.com/user-attachments/assets/5f881134-0a02-436a-85e2-a5344f87a038)